docs/guide-using-an-ide-for-backend.md

```diff
@@ -1,21 +1,20 @@
-# Using an IDE (Intellij)
+# Using an IDE (Intellij) for development of MOLGENIS backend
 
-A rough guide on how to develop molgenis in IntelliJ IDEA is described below.
-This was created using IntelliJ IDEA 2016.1 Ultimate, courteously provided to us by
-JetBrains on an open source license. Most of the time we use the latest IntelliJ version.
+A rough guide on how to develop MOLGENIS in IntelliJ IDEA is described below.
+This was created using IntelliJ IDEA Ultimate. Most of the time we use the latest IntelliJ version.
 
```

Comment:
This section should also provide a migration guide: you should stop local services, etc.